addons/zotero/migrations/0005_auto_20180216_0849.py

```diff
@@ -0,0 +1,68 @@
+# -*- coding: utf-8 -*-
+# Generated by Django 1.11.9 on 2018-02-16 14:49
+from __future__ import unicode_literals
+
+from bulk_update.helper import bulk_update
+from django.db import migrations
+
+def reverse_func(state, schema):
+    print 'Starting reverse zotero library migration'
+    modify_node_settings(state, None)
+    modify_user_settings(state, False, None)
+
+def modify_node_settings(state, library_name):
+    """
+    Updates the library_id for all ZoteroNodeSettings
+    """
+    ZoteroNodeSettings = state.get_model('addons_zotero', 'NodeSettings')
+    ZoteroNodeSettings.objects.all().update(library_id=library_name)
+    print 'NodeSettings Updated'
```

Comment:
Use `logging` for logging in migrations. We can probably just omit these though.